certificate-profile-subordinate-CA.md

```diff
@@ -19,7 +19,7 @@
 | keyUsage  | Mandatory | True | Bit positions for keyCertSign and cRLSign shall be set. <br> If the Subordinate CA Private Key is used for signing OCSP responses, then the digitalSignature bit shall also be set. |
 | extkeyUsage |  Mandatory  | False | This extension is required for Technically constrained nameConstraints per Section 7.1.2.2 and Section 7.1.5. <br> Required Extended Key Usage: <br> Server Authentication id-kp-serverAuth {1.3.6.1.5.5.7.3.1} <br><br> Optional Extended Key Usage: <br> Client Authentication id-kp-clientAuth {1.3.6.1.5.5.7.3.2} <br>id-kp-OCSPSigning {1.3.6.1.5.5.7.3.9} <br> Other values may be present consistent with use for server authentication, with approval by the FPKIPA. |
 | certificatePolicies |  Mandatory  | False | See Section 7.1.6.3. At least one US Government certificate policy OID listed in Section 7.1.6.1 asserting compliance with this CP, and one CAB Forum certificate policy OID listed in Section 7.1.6.1 asserting compliance with the CAB Forum Baseline Requirements. The certificate shall include all the certificate policy OIDs for all certificates issued by the CA.  |
-| subjectAltName | Optional | False  |  |
+| subjectAltName | Optional | False  | Underscore characters (“_”) shall not be present in dNSName entries. |
```

Comment:
Let's rethink the way it's written.  It could be be misconstrued as dNSName entries should be in the SAN in a subordinate CA cert...if SAN is included.

Reply:
I read it as if the SAN is included and if a dnsName is used then it shall not include an underscore.

Reply:
I get it...I was looking at our issues and notes on why we even made a SAN optional for a CA cert.  This might be another "remove it entirely as even an _option_"